File: tests/mobi/hsz/idea/gitignore/inspections/UnusedEntryInspectionTest.java
package mobi.hsz.idea.gitignore.inspections;

import mobi.hsz.idea.gitignore.codeInspection.IgnoreUnusedEntryInspection;

public class UnusedEntryInspectionTest extends InspectionTestCase {
    @Override
    public void setUp() throws Exception {
        super.setUp();
        myFixture.enableInspections(IgnoreUnusedEntryInspection.class);
    }

    public void testUnusedFile() throws Exception {
//        doHighlightingTest();
    }

    public void testUnusedDirectory() throws Exception {
        doHighlightingTest();
    }
}

File: src/test/kotlin/mobi/hsz/idea/gitignore/inspections/UnusedEntryInspectionTest.kt
// Copyright 2000-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.

package mobi.hsz.idea.gitignore.inspections

import mobi.hsz.idea.gitignore.codeInspection.IgnoreUnusedEntryInspection

class UnusedEntryInspectionTest : InspectionTestCase() {

    @Throws(Exception::class)
    public override fun setUp() {
        super.setUp()
        myFixture.enableInspections(IgnoreUnusedEntryInspection::class.java)
    }

    @Throws(Exception::class)
    fun testUnusedFile() {
        doHighlightingTest()
    }

    @Throws(Exception::class)
    fun testUnusedDirectory() {
        doHighlightingTest()
    }
}
